### PR TITLE
Form data update

### PR DESCRIPTION
**Pull request by remiriv**
## Proposed changes

---

- Allows the anim to update data and to see which data are not credible 

## Types of changes

---

What types of changes does your code introduce to ogre ?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Other (please describe):

## Checklist

---

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

offseason_ogre/blob/master/CONTRIBUTING.md) doc

- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation and modify the README.md (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] If the feature changes current behavior, reasons why your solution is better.

## What is the current behavior?

---

-
-
-

## What is the new behavior?

---

-
-
-

## Further comments

---

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

## Where has this been tested?

---

Docker version: …
Operating System: …
node version: …
postgresql version: …

### CODE REVIEW

File: packages/client/src/modules/administration/Games/Game/FormVerification.tsx [hunk 1]
```diff
@@ -21,6 +39,7 @@ function FormVerification({
   openFormValidation: boolean;
   setOpenFormValidation: (value: boolean) => void;
 }): JSX.Element {
+  const [updatedRows, setUpdatedRows] = useState<FormattedRow[]>([]);
```
Comment (louis-sanna-dev) [inline packages/client/src/modules/administration/Games/Game/FormVerification.tsx:44]: @remiriv à quoi sert cette état?

Comment (remiriv) [reply packages/client/src/modules/administration/Games/Game/FormVerification.tsx:44]: C'est pour stocker les lignes qu'on a modifié tant qu'on a pas appuyé sur sauvegarder les modifications et que ça a pas envoyé en base 
Comme ça si on clique sur annuler ça annule et ça revient aux données précédentes mais on peut tout envoyer en base d'un coup en sauvegardant

File: packages/client/src/modules/persona/consumption/utils.ts [hunk 2]
```diff
@@ -32,7 +32,11 @@ export const getHeatingConsumption = (
 
 export const getHeatingConsumptionInvoiceCoeff = (profile: PersoForm) => {
   const { heatingInvoice, heatingEnergy } = profile;
-  return heatingInvoice / getHeatingEnergyCoeff(heatingEnergy);
+  return (
```
Comment (Baboo7) [inline packages/client/src/modules/persona/consumption/utils.ts]: Le fait de tronquer la valeur en passant par une string puis en convertissant à nouveau la valeur en nombre va introduire des erreurs dans les calculs qui seront difficiles à debugger. Il vaut mieux faire tous les calculs avec des flottants et convertir en string seulement pour l'affichage aux endroits où on en a besoin

File: packages/client/src/modules/play/Personalization/models/form.tsx [hunk 3]
```diff
@@ -124,6 +124,7 @@ export interface Question {
   valueType: string;
   options?: DropdownOption[];
   defaultValue?: boolean | string | number;
+  credibilityConditions?: Condition[];
```
Comment (Baboo7) [inline packages/client/src/modules/play/Personalization/models/form.tsx:127]: Très clair le nommage 👍

File: packages/client/src/modules/persona/consumption/utils.ts (hunk 2 above)
Comment (remiriv) [reply packages/client/src/modules/persona/consumption/utils.ts]: Effectivement bien vu !

**Comment by remiriv** [comment]
@Baboo7 les modifs sont push !

**Pull request merged**